src/include/storage/recovery/replication_log_provider.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <memory>
+#include "storage/recovery/abstract_log_provider.h"
+
+namespace terrier::storage {
+
+/**
+ * TODO(Tianlei/Gus): Replace this file with Gus's version
+ */
+class ReplicationLogProvider {
```

Comment:
@GustavoAngulo This is just a basic frame I created. I am not sure if this is sufficient.